tests/integration/client_integration.py

```diff
@@ -1,7 +1,7 @@
 """Integration tests for client library"""
 from hil.flaskapp import app
 from hil.client.base import ClientBase, FailedAPICallException
-from hil.errors import BadArgumentError, UnknownSubtypeError
+from hil.errors import BadArgumentError
```

Comment:
looks like we don't use UnknownSubtypeError anywhere now. Let's just delete it while we are at it.